modules/configuring-output-format.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+:_content-type: CONCEPT
+[id="configuring-output-format_{context}"]
+= Configuring output format
+
+When you check policy compliance by using the `roxctl deployment check`, `roxctl image check`, or `roxctl image scan` commands, you can specify the output format by using the `-o` option. This option determines how the output of a command is displayed in the terminal.
```

Comment:
While output format can be changed for all three commands, I'd drop the `roxctl image scan` command here since it does not check for policy compliance, but instead focuses on scanning.